desktop_style.css

```diff
@@ -0,0 +1,458 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+@media (min-width: 768px) {
+  .mobile {
+    display: none;
+  }
+
+  button:hover {
+    cursor: pointer;
+    background: #6070ff;
+  }
+
+  button:active {
+    background: #2230d2;
+    transform: translateY(4px);
+  }
+
+  button:disabled {
+    background: #e5e5e5;
+  }
+
+  main {
+    width: 100vw;
+    background: #e5e5e5;
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: auto auto auto;
+  }
+
+  .intro_page {
+    height: 100vh;
+    background: #fff;
+  }
+
+  .navbar {
+    display: flex;
+    height: 72px;
+    padding: 26px 10.763vw 26px 10.763vw;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  nav {
+    display: none;
+  }
+
+  header a {
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+  }
+
+  .menu_bar {
+    display: flex;
+    column-gap: 2.222vw;
+  }
+
+  .menu_bar li {
+    list-style: none;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .background_content {
+    margin-top: 46px;
+    width: 97.04vw;
+    padding-left: 2.402vw;
+    padding-right: 0.555vw;
+    background-image: url(./images/Header.svg);
+    height: 721.46px;
+  }
+
+  .content {
+    padding-left: 18.055vw;
+    padding-right: 18.055vw;
+  }
+
+  .content h1 {
+    padding-top: 241px;
+    font-weight: 700;
+    font-size: 49px;
+    line-height: 52px;
+    color: #091e42;
+    font-style: normal;
+  }
+
+  .content p {
+    margin-top: 12px;
+    margin-bottom: 12px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .content h6 a {
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .social-media-icons {
+    margin-top: 12px;
+    display: flex;
+    column-gap: 1.388vw;
+    align-items: center;
+  }
+
+  .social-media-icons li {
+    list-style: none;
+  }
+
+  .Works_Section {
+    width: 100vw;
+    background: #fff;
+    display: grid;
+    grid-template-columns: 80%;
+    grid-template-rows: 24% 24% 24% 24%;
+  }
+
+  .card {
+    display: flex;
+    margin-left: 11.52vw;
+    margin-right: 11.52vw;
+    width: 80.277vw;
+    column-gap: 1.527vw;
+    height: 496px;
+    margin-bottom: 142px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    padding-top: 114px;
+    padding-left: 1.5vw;
+    padding-right: 1.5vw;
+  }
```

Comment:
- [ ] The screenshots are not contained within the cards in the works section. Kindly fix this issue and your project is good to go.  Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/178108010-4db6a654-b977-4f7c-ad4b-4fddd74c648c.png)  |  ![image](https://user-images.githubusercontent.com/60618877/178108033-30b4f60a-f196-47d0-ae5d-8f2a3b3ee283.png)

Reply:
Hey Sumairq, thank you for the code review. I just wanted to clarify something real quick before I proceed, is your review comment more like a suggestion that I should contain the screenshots within the cards in the works section, or do you mean the screenshots should not be in the container within the cards in the works section. I'm a little bit confused, please may you clarify this for me. I have attached a screenshot of the code.
 ![image](https://user-images.githubusercontent.com/97179349/178108621-898278aa-7828-494a-9211-663b33fcd804.png)
As of now, the screenshot images in my code are contained within the "card" container in the works section, so I'm assuming you mean I should remove them out of the "card" container? Is that right?